catalog/pom.xml

```diff
@@ -48,6 +48,7 @@
         <jaxb.annotate.plugin.version>0.6.0</jaxb.annotate.plugin.version>
         <jaxb.visitor.plugin.version>1.13</jaxb.visitor.plugin.version>
         <catalog.joda.time.version>2.2</catalog.joda.time.version>
+        <tika.thirdparty.bundle.version>1.0</tika.thirdparty.bundle.version>
```

Comment:
I missed this on the thirdparty PR - it is a typical convention to use the same version number of the project we are wrapping and then adding to the version number (basically `1.13.0_1` instead of `1.0`) since we are just repackaging it and not changing the underlying code.  this will also help developers know which version of the API's they should refer to when adding new capabilities.